ngraph/python/src/ngraph/opset8/ops.py

```diff
@@ -276,3 +276,33 @@ def gather(
         "batch_dims": batch_dims
     }
     return _get_node_factory_opset8().create("Gather", inputs, attributes)
+
+
+@nameable_op
+def random_uniform(
+        output_shape: NodeInput,
+        min_val: NodeInput,
+        max_val: NodeInput,
+        output_type: str,
+        global_seed: int = 0,
+        op_seed: int = 0
```

Comment:
Is `op_seed` name a planned change? Is there a check in C++ implementation that ensure integers are positive? Moreover there is no info about default value.

Docs spec:
https://github.com/openvinotoolkit/openvino/blame/master/docs/ops/generation/RandomUniform_8.md#L185-L190